lib/app/utils/hl7_validator.rb

```diff
@@ -6,7 +6,9 @@ class HL7Validator
     ISSUE_DETAILS_FILTER = [
       %r{^Sub-extension url 'introspect' is not defined by the Extension http://fhir-registry\.smarthealthit\.org/StructureDefinition/oauth-uris$},
       %r{^Sub-extension url 'revoke' is not defined by the Extension http://fhir-registry\.smarthealthit\.org/StructureDefinition/oauth-uris$},
-      /^URL value .* does not resolve$/
+      /^URL value .* does not resolve$/,
+      /^vs-1: if Observation.effective\[x\] is dateTime and has a value then that value shall be precise to the day/, # Invalid invariant in FHIR v4.0.1
+      /^uscore-1: if Observation.effective\[x\] is dateTime and has a value then that value shall be precise to the day/ # Invalid invariant in US Core v3.1.1
```

Comment:
These periods should technically be escaped like in the filters above, but I don't think it really matters as it's difficult to imagine getting false positives from not escaping them.